docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/link/link.twig

```diff
@@ -18,11 +18,10 @@
 #}
 
 {% set theme_class = 'civictheme-theme-%s'|format(theme|default('light')) %}
-{% set is_external = is_external ? 'civictheme-link--external' : '' %}
```

Comment:
please reinstate the class - we still want to know if this is external or not. even if this class does not have any rules